<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget
{
    background-color: #6357B7;
    border-radius: 0px 0px 15px 15px;
}

QPushButton
{
    color: white;
    background-color: #6357B7;
    border-radius: 15px;
    font: 16pt &quot;Bahnschrift&quot;;
}

QPushButton:disabled
{
    color: #6357B7;
    background-color: white;
    border-radius: 0px;
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="RegistrarAvion">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Registrar Avion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="EliminarAvion">
        <property name="text">
         <string>Eliminar Avion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DatosAvion">
        <property name="text">
         <string>Datos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ReporteAvion">
        <property name="text">
         <string>Reportes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget
{
    background-color: white;
}

QPushButton
{
    color: white;
    background-color: #6357B7;
    border-radius: 15px;
    font: 14pt &quot;Bahnschrift&quot;;
}

QPushButton:Pressed
{
    background-color: #C397E4;
}

QPushButton:disabled
{
    color: #6357B7;
    background-color: white;
    border-radius: 0px;
}

QLineEdit
{
    color: #6357B7;
    background-color: white;
    border : 2px solid #6357B7;
    border-radius: 15px;
    font: 10pt &quot;Bahnschrift&quot;;
    border: 1px solid #6357B7;
    padding: 10px;
}

QLabel
{
    background-color: transparent;
    color: #6357B7;
    font: 12pt &quot;Bahnschrift&quot;;
}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
